pandas/core/arrays/numpy_.py

```diff
@@ -398,6 +402,10 @@ def arithmetic_method(self, other):
 
     _create_comparison_method = _create_arithmetic_method
 
+    # ------------------------------------------------------------------------
+    # String methods interface
+    _str_na_value = np.nan
```

Comment:
not pd.NA?

Reply:
This is for `object` dtype strings (the default), not `string` dtype, so those are still using np.nan (or None)

Reply:
And StringArray overrides it to be pd.NA